mllib/src/test/scala/org/apache/spark/ml/classification/DecisionTreeClassifierSuite.scala

```diff
@@ -232,6 +232,13 @@ class DecisionTreeClassifierSuite extends SparkFunSuite with MLlibTestSparkConte
     compareAPIs(rdd, dt, categoricalFeatures = Map.empty[Int, Int], numClasses)
   }
 
+  test("copied model must have the same parent") {
```

Comment:
I think it is better to make helper method in order not to make duplicates. But it is better to have one unit test has one assert to make it easy to specify which test fail. How about?